fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb

```diff
@@ -22,7 +22,7 @@
       it "uploads dSYM files" do
         binary_path = './spec/fixtures/screenshots/screenshot1.png'
         dsym_path = './spec/fixtures/dSYM/Themoji.dSYM'
-        gsp_path = './spec/fixtures/plist/Info.plist'
+        gsp_path = './spec/fixtures/plist/With Space.plist'
```

Comment:
This test already covers `shellescape` so just switched the path here.

Reply:
🙌